### PR TITLE
Addresses CVE-2024-4068 and updates yarn.lock

### PR DESCRIPTION
**Pull request by DarshitChanpura**
### Description
Updates yarn.lock
Addresses CVE-2024-4068

### Category
Maintenance

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@hapi/cryptiles": "5.0.0",
     "@hapi/wreck": "^17.1.0",
+    "debug": "^4.3.4",
```
Comment (DarshitChanpura) [inline package.json:46]: yarn upgrade auto-added this entry

**Review by scrawfor99 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (derek-ho) [reply package.json:46]: I don't think this is right. We are not using debug, it is a transitive dependency, so it shouldn't be added here. We already have a resolution for it.

**Review by derek-ho — CHANGES_REQUESTED** [review]
I don't think this is right. We are not using debug, it is a transitive dependency, so it shouldn't be added here. We already have a resolution for it.

File: yarn.lock [hunk 2]
```diff
@@ -2257,6 +2257,11 @@ glob-parent@^3.1.0, glob-parent@^5.1.2, glob-parent@~5.1.2:
   dependencies:
     is-glob "^4.0.1"
 
+glob-to-regexp@^0.4.1:
```
Comment (derek-ho) [inline yarn.lock:2260]: I don't see braces version change here. Can you re-run `yarn`?

Comment (DarshitChanpura) [reply yarn.lock:2229]: done.

**Comment by derek-ho** [comment]
@DarshitChanpura can we update the title/description of this PR that we are addressing the braces CVE?

**Comment by DarshitChanpura** [comment]
@derek-ho Anything blocking this PR from getting merged?

**Comment by derek-ho** [comment]
> @derek-ho Anything blocking this PR from getting merged?

Nope it looks good to me now

**Review by derek-ho — APPROVED** [review]

**Review by cwperks — APPROVED** [review]

**Pull request merged**